src/sentry/static/sentry/app/utils/discover/eventView.tsx

```diff
@@ -84,16 +85,6 @@ export function isFieldSortable(field: Field, tableMeta?: MetaType): boolean {
   return !!getSortKeyFromField(field, tableMeta);
 }
 
-const generateFieldAsString = (col: Column): string => {
```

Comment:
Needed this in the `MetricField` so pulled it into the fields module.

Reply:
Thanks this function has been in the wrong place for a while now. Good to see it moved in with its friends.